Proofread docs, remove links from index.rst (#18)
tarantool/tarantool-internals: source/replication.rst

We allocate new replica and assign an applier to it.

Note that replica state is driven by ``replica_on_applier_state_f``
trigger. We won't be jumping into it right now but the important thing is
that this trigger sends ``fiber_cond_signal(&replicaset.applier.cond)``
to the main replicaset instance.

> Now back to the caller, i.e. ``applier_sync_state``. The replica
> instances are created we continue walking over appliers

Reviewer: ```suggestion
Now back to the caller, which is the ``applier_sync_state``.
The replica instances are created and we continue walking over appliers:
```

        ...
        rlist_swap(&replicaset.anon, &anon_replicas);

> At the end the anonymous replicas (which is not connected)

Reviewer: ```suggestion
At the end the anonymous replicas (which are not connected)
```
